Watchman.Discord/ReactionsService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Watchman.Discord
+{
+    public static class ReactionsService
```

Comment:
czemu mamy statyczny serwis?

Reply:
żeby wszędzie był łatwy dostęp do niego, aby każdy mógł coś do niego w dowolnym momencie dodać. Gdyby był zwykłą klasą to ciężko byłoby zapewnić wszędzie dostęp do jego instancji.

Reply:
od tego jest IoC (inversion of control - u nas tym narzędziem jest AutoFac)
IoC zajmuje się wstrzyknięciem zależności do obiektów
czyli u nas przez ctor wchodzi ReactionsService i nie trzeba się przejmować tworzeniem obiektu, bo robi to za nas AutoFac (mamy ustawione, że z każdej klasy tworzona jest tylko jedna instancja)
Podsumowując, zrób tą klasę jako niestatyczną, a kiedy trzeba z niej skorzystać po prostu doda się ją jako argument ctora

Reply:
czyli wystarczy, że zamienię na zwykłą klasę i to wystarczy?

Reply:
w pierwszej kolejności powinieneś się zastanowić jak to powinno działać i jaki jest twój cel
bo robienie serwisu który tylko przekazuje func z workflow nie ma za bardzo sensu 
więcej kombinowania przy tym będzie niż korzyści

Reply:
Wcześniej serwis robił więcej, ale ty i mateusz chcieliście, aby część jego funkcjonalności przenieść gdzie indziej i tak właśnie zrobiłem.